perftest/datanode.go

```diff
@@ -17,6 +17,23 @@ type dnWrapper struct {
 	wallet   walletWrapper
 }
 
+func (d *dnWrapper) getNetworkParam(param string) (string, error) {
+	request := &datanode.NetworkParametersRequest{}
+
+	response, err := d.dataNode.NetworkParameters(context.Background(), request)
```

Comment:
not a big deal but why use datanode v1? (line10) v2 has GetNetworkParameter() for a single param.

Reply:
Thanks for the tip, I'll update now.